R/extract.R

```diff
@@ -5354,3 +5354,91 @@ extract.feglm <- function(model, include.deviance = TRUE, include.nobs = TRUE,
     return(tr)
 }
 setMethod("extract", signature = className("feglm", "alpaca"), definition = extract.feglm)
+                                         
+                                         # extension for gnm objects (gnm package)
```

Comment:
What happened to the spacing here?

Reply:
I have no idea... I guess it's related to the conflicts...